<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>debug_signal</class>
 <widget class="QWidget" name="debug_signal">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLineEdit" name="le_9">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>290</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_6">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>170</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lb_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>250</y>
     <width>20</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Agency FB</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>3</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lb_1">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>170</y>
     <width>20</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Agency FB</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>1</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="pB_Exit">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>410</y>
     <width>80</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>返回</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_3">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>250</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_20">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>330</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_14">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>290</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_5">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>330</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_10">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>330</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_11">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>170</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lb_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>290</y>
     <width>20</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Agency FB</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>4</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lb_5">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>330</y>
     <width>20</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Agency FB</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>5</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_1">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>170</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_8">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>250</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_2">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>210</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_17">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>210</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_4">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>290</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_15">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>330</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_19">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>290</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_12">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>210</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_13">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>250</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_18">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>250</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_7">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>210</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lb_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>210</y>
     <width>20</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Agency FB</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>2</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_16">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>170</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QRadioButton" name="rb_led1_405">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>210</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>405 1</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="rb_led2_405">
   <property name="geometry">
    <rect>
     <x>559</x>
     <y>254</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>405 2</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="rb_led3_405">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>290</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>405 3</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="rb_led4_405">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>330</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>405 4</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="rb_led5_405">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>370</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>405 5</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="rb_led5_610">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>170</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>610 5</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="rb_led1_610">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>10</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>610 1</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="rb_led2_610">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>50</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>610 2</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="rb_led3_610">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>90</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>610 3</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="rb_led4_610">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>130</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>610 4</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_22">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>210</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_24">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>290</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_23">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>250</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_25">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>330</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_21">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>170</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QCheckBox" name="CB_TOL">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>20</y>
     <width>171</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="text">
    <string>总大肠</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="CB_FEC">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>80</y>
     <width>171</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="text">
    <string>耐热大肠</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="CB_TPC">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>20</y>
     <width>171</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="text">
    <string>细菌总数</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="CB_E">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>80</y>
     <width>171</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="text">
    <string>埃希氏</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="rb_led5_365">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>170</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>365 5</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="rb_led2_365">
   <property name="geometry">
    <rect>
     <x>559</x>
     <y>54</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>365 2</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="rb_led1_365">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>10</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>365 1</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="rb_led3_365">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>90</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>365 3</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="rb_led4_365">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>130</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>365 4</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="CB_Debug">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>310</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>LED_DEBUG</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
